package.json

```diff
@@ -10,11 +10,18 @@
   "author": "David Boram Hong",
   "license": "ISC",
   "dependencies": {
+    "bcrypt": "^5.1.1",
     "cors": "^2.8.5",
     "dotenv": "^16.4.5",
     "express": "^4.18.2",
+    "express-flash": "^0.0.2",
+    "express-session": "^1.18.0",
```

Comment:
## Improvement
As these dependencies are not going to be used, removing them will be better for resource management.

Reply:
Improvement has been made.